Module not found: Error: Cannot resolve module 'ng-webworker'

Occurs when doing `require("ng-webworker");`. A more detailed error:

> Module not found: Error: Cannot resolve module 'ng-webworker' in /Users/Nddt/web-app/app/components
resolve module ng-webworker in /Users/Nddt/web-app/app/components
looking for modules in /Users/Nddt/web-app/node_modules
resolve 'file' ng-webworker in /Users/Nddt/web-app/node_modules
resolve file
/Users/Nddt/web-app/node_modules/ng-webworker is not a file
/Users/Nddt/web-app/node_modules/ng-webworker.webpack.js doesn't exist
/Users/Nddt/web-app/node_modules/ng-webworker.web.js doesn't exist
/Users/Nddt/web-app/node_modules/ng-webworker.js doesn't exist
/Users/Nddt/web-app/node_modules/ng-webworker.json doesn't exist
resolve 'file' or 'directory' /Users/Nddt/web-app/node_modules/ng-webworker
resolve file
/Users/Nddt/web-app/node_modules/ng-webworker is not a file
/Users/Nddt/web-app/node_modules/ng-webworker.webpack.js doesn't exist
/Users/Nddt/web-app/node_modules/ng-webworker.web.js doesn't exist
/Users/Nddt/web-app/node_modules/ng-webworker.js doesn't exist
/Users/Nddt/web-app/node_modules/ng-webworker.json doesn't exist
resolve directory
directory default file index
resolve file index in /Users/Nddt/web-app/node_modules/ng-webworker
/Users/Nddt/web-app/node_modules/ng-webworker/index doesn't exist
/Users/Nddt/web-app/node_modules/ng-webworker/index.webpack.js doesn't exist
/Users/Nddt/web-app/node_modules/ng-webworker/index.web.js doesn't exist
/Users/Nddt/web-app/node_modules/ng-webworker/index.js doesn't exist
/Users/Nddt/web-app/node_modules/ng-webworker/index.json doesn't exist
[/Users/Nddt/web-app/node_modules/ng-webworker.webpack.js]
[/Users/Nddt/web-app/node_modules/ng-webworker.webpack.js]
[/Users/Nddt/web-app/node_modules/ng-webworker.web.js]
[/Users/Nddt/web-app/node_modules/ng-webworker.web.js]
[/Users/Nddt/web-app/node_modules/ng-webworker.js]
[/Users/Nddt/web-app/node_modules/ng-webworker.js]
[/Users/Nddt/web-app/node_modules/ng-webworker.json]
[/Users/Nddt/web-app/node_modules/ng-webworker.json]
[/Users/Nddt/web-app/node_modules/ng-webworker/index]
[/Users/Nddt/web-app/node_modules/ng-webworker/index.webpack.js]
[/Users/Nddt/web-app/node_modules/ng-webworker/index.web.js]
[/Users/Nddt/web-app/node_modules/ng-webworker/index.js]
[/Users/Nddt/web-app/node_modules/ng-webworker/index.json]
@ ./app/components/app.js 23:4-27

--- a/package.json
+++ b/package.json
@@ -12,6 +12,7 @@
     "type": "git",
     "url": "https://github.com/mattslocum/ng-webworker.git"
   },
+  "main": "src/ng-webworker.js",
   "devDependencies": {
     "gulp": "^3.9.1",
     "gulp-rename": "^1.2.2",

